### PR TITLE
[DOC-2261] Update delegate auto upgrade status info, add delegate list page doc

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update delegate auto upgrade status info, add delegate list page doc

For reviewers, previews are available:

[Delegate overview](https://6580eb8658f3e6411295d9ff--harness-developer.netlify.app/docs/platform/delegates/delegate-concepts/delegate-overview)
[Delegate automatic upgrades and expiration policy](https://6580eb8658f3e6411295d9ff--harness-developer.netlify.app/docs/platform/delegates/install-delegates/delegate-upgrades-and-expiration)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/delegates/install-delegates/delegate-upgrades-and-expiration.md [hunk 1]
```diff
@@ -132,13 +132,17 @@ spec:
 
 ### Determine if automatic upgrade is enabled
 
-When a delegate is installed, it may take up to an hour to determine if the `upgrader` was removed during installation. During that time, the delegate shows a status of **DETECTING**. 
+When a delegate is installed, it may take up to an hour by default to determine if the `upgrader` was removed during installation. During that time, the delegate shows a status of **DETECTING**.
 
-To find the delegate status, select an account, a project, or an organization, and then select **Delegates**. 
+Harness updates the status when `upgrader` is makes it’s first API call to the Harness platform. The default schedule is one hour, but the schedule is configurable. If Harness doesn't detect the upgrader API call within 90 minutes, the upgrade status is updated from `DETECTING` to `OFF`.
```
Comment (mdmilic) [inline docs/platform/delegates/install-delegates/delegate-upgrades-and-expiration.md:137]: `is` is probably a typo in: when `upgrader` is makes

Comment (brian-f-harness) [inline docs/platform/delegates/install-delegates/delegate-upgrades-and-expiration.md:137]:
```suggestion
Harness updates the status when `upgrader` makes it’s first API call to the Harness platform. The default schedule is one hour, but the schedule is configurable. If Harness doesn't detect the upgrader API call within 90 minutes, the upgrade status is updated from `DETECTING` to `OFF`.
```

**Review by mdmilic — APPROVED** [review]
looks good

**Comment by brian-f-harness** [comment]
> looks good

Thanks for all the details and for reviewing @mdmilic.

**Pull request merged**